lib/stream.js

```diff
@@ -26,14 +26,10 @@ const getBufferedData = async (streamPromise, encoding) => {
 };
 
 const getStdioPromise = ({stream, stdioStreams, encoding, buffer, maxBuffer}) => stdioStreams[0].direction === 'output'
-	? getStreamPromise({stream, encoding, buffer, maxBuffer, objectMode: stream?.readableObjectMode})
+	? getStreamPromise({stream, encoding, buffer, maxBuffer})
 	: undefined;
 
-const getAllPromise = ({spawned, encoding, buffer, maxBuffer}) => {
-	const stream = getAllStream(spawned, encoding);
-	const objectMode = spawned.stdout?.readableObjectMode || spawned.stderr?.readableObjectMode;
```

Comment:
`spawned.all.readableObjectMode` (that is, `stream.readableObjectMode`) is now the same as `spawned.stdout?.readableObjectMode || spawned.stderr?.readableObjectMode` with the new version of `merge-streams`.